sklearn/linear_model/_huber.py

```diff
@@ -205,7 +205,7 @@ class HuberRegressor(LinearModel, RegressorMixin, BaseEstimator):
     >>> y[:4] = rng.uniform(10, 20, 4)
     >>> huber = HuberRegressor().fit(X, y)
     >>> huber.score(X, y)
-    -7.284608623514573
+    -7.284608623514572
```

Comment:
Why this change? Maybe we are better using an ellipsis?

Reply:
I'm not really sure honestly. I double checked and couldn't find a change that could have caused this. It was only failing on a specific platform (MacOS I think), so I don't think it's related to the changes. I'll use ellipsis

Reply:
OK. I agree with you.